source/src_pw/potential.cpp

```diff
@@ -34,7 +34,6 @@ Potential::~Potential()
 void Potential::allocate(const int nrxx)
 {
     ModuleBase::TITLE("Potential", "allocate");
-    assert(nrxx > 0);
 
     delete[] this->vltot;
     this->vltot = new double[nrxx];
```

Comment:
Won't an error be returned during allocating vltot when nrxx <  0 ?

Reply:
Maybe I should change it to assert(nrxx >= 0)

Reply:
> Maybe I should change it to assert(nrxx >= 0)

Please do not use `assert` for it - use if-statement instead.